apps/cloudinary/src/index.js

```diff
@@ -76,8 +76,13 @@ function renderDialog(sdk) {
   const instance = cloudinary.createMediaLibrary(options, {
     insertHandler: data => sdk.close(data)
   });
+  const show_options={};
```

Comment:
Minor nit pick but let's keep style inline with the rest of the code.

```suggestion
  const showOptions = {};
```